packages/core/src/lib/TwaManifest.ts

```diff
@@ -304,6 +306,8 @@ export interface TwaManifestJson {
   generatorApp?: string;
   webManifestUrl?: string;
   fallbackType?: FallbackType;
+  appsFlyer?: boolean;
+  plugins?: string[];
```

Comment:
Why both of these? Surely we just need one?

Reply:
Indeed. Removed `appsFlyer`.